### DISCUSSION TITLE
CMU 15-251 理论计算机科学的伟大思想 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#60] [Announcements]
# CMU 15-251 理论计算机科学的伟大思想 | HackWay技术学习路线

课程名称： Structure and Interpretation of Computer Programs

https://hackway.org/docs/cs/junior/reasoning/cs15251

**Comment by TsingBochun** [comment]
课程名称： Structure and Interpretation of Computer Programs，貌似写错了？

**Reply by lidongyx** [reply]
嗯，我周末修改一下
